backend/models/declarations/group.ts

```diff
@@ -0,0 +1,31 @@
+import * as Mongoose  from 'mongoose';
+import { WeekScheduleDoc } from '@models/week-schedule';
+import { ExamDoc } from '@models/exam';
+import { Maybe } from '@modules/interfaces';
+
+/**
+ * This model represens group of students with schedule etc
+ */
+export interface GroupData {
+    name_grop:           string; // use as a name of group chat, where bot was added
+    group_id:            number; // use as a chat id of group chat
+    num_of_participants: number; // ? store a num of all particepents (or make method and fetch all particepents to caluclate (sh*t))
+    date_of_created:     Date;   // date of first usage /enroll in this group 
+    // term:                number; // current term
+    // from KPSessionBotArchitecture  
+    exams?:              Maybe<ExamDoc[]>;   
+    week_schedule?:      Maybe<WeekScheduleDoc>;
```

Comment:
`WeekScheduleDoc` already has nullable `WeekScheduleDoc.subjects` field, if you want to we the whole subdocument nullable then you need to refactor `WeekScheduleDoc`.